packages/ui-kit/src/components/content-notifications.tsx

```diff
@@ -82,21 +84,26 @@ const ContentNotification = (props) => {
 
           </Info>
         */}
-        <TypographyContainer>{props.children}</TypographyContainer>
+        <TypographyContainer>{children}</TypographyContainer>
       </SpacingsInline>
     </Container>
   );
 };
-ContentNotification.propTypes = {
-  type: PropTypes.oneOf(['error', 'warning', 'info']).isRequired,
-  children: PropTypes.node.isRequired,
-};
-ContentNotification.defaultProps = {
```

Comment:
Please define `defaultProps`. Example how to use it: https://github.com/commercetools/ui-kit/blob/f1bdecca464f73e94edc6b758ff271cb81695a3e/packages/components/avatar/src/avatar.tsx#L46-L54

Reply:
My understanding was that defaultProps are already on the deprecation path and that the object spread defaults I use above are the way to go now (and much more concise and javascript-ish :-) )

https://github.com/yannickcr/eslint-plugin-react/issues/2396